angular.json

```diff
@@ -122,6 +122,123 @@
           }
         }
       }
+    },
+    "@enigmatry/docs": {
```

Comment:
Maybe we should rename project name to something like @enigmatry/docs-app (just project name, nothing else)
so it's clear when building with cli: ng build @enigmatry/docs-app